Rakefile

```diff
@@ -4,7 +4,3 @@
 require_relative 'config/application'
 
 Rails.application.load_tasks
-
```

Comment:
Cool - lines 8-10 were from a stack overflow suggestion, I'm glad we don't need them